backend/src/emissions/schema.py

```diff
@@ -43,7 +43,7 @@
 from graphql_jwt.decorators import login_required
 import warnings
 
-warnings.filterwarnings("error")
+# warnings.filterwarnings("error")
```

Comment:
what does this do? if it's commented, can we just delete it?

Reply:
this treats warnings as errors, so instead of just logging a warning message, enabling this will throw an error and the program stops. We had to disable it again because the co2calculator throws warnings when default values are used if the user doesn't provide any data.